src/server.jsx

```diff
@@ -132,6 +132,9 @@ function setupServer(req, res, next) {
     // Displays error in console
     console.error(error);
 
+    // If error happens in Volto code itself
+    if (!error.status) res.status(500);
+
     res
       .status(error.status)
```

Comment:
```suggestion
    // If error happens in Volto code itself: 500
    res
      .status(error.status || 500)
```

Reply:
Ok, yeah, I will correct it, I also think it's better.